MIP/mip-40/README.md

```diff
@@ -0,0 +1,83 @@
+# MIP-40: *AB-FFS Decoupled*
+- **Description**: Introduces architectural guide for ***AB-FFS Decoupled***. 
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Reviewer**: [Andreas Penzkofer](mailto:andreas.penzkofer@movementlabs.xyz)
+- **Desiderata**: [MD-38](https://github.com/movementlabsxyz/MIP/pull/38)
+
+## Abstract
+
+This MIP introduces the architectural guide for implementing ***AB-FFS Decoupled***, a means of using the Atomic Bridge and Fast-finality Settlement with a fixed supply of token. This decoupled approach relies on burning gas on the L2 into an L2 pool and issuing rewards in an LP token on the L1. Details of the L2 pool recirculation and the L1 LP token issuance are largely left to follow-on MIPs, however we do specify general constraints and guidelines. 
+
+We also propose the usage of intermediary tokens for bridging to fully decouple the L1 and L2 systems. This allows for the L2 pool to be recirculated without the need for a direct bridge to the L1 and exposure to fallibility of the bridge on either side. 
```

Comment:
We do not need intermediary tokens. 
The mechanics of charging fees on L2 and rewarding validators on L1 are independent from the bridge. 
On L1, you can do whatever you want with the ERC-20 MOVE, it is not bound to rewarding validators.
On L2, it is the same, it is the wrapped token that users can exchange if they want, and one usage is to pay for transaction fees.

Reply:
1. We can't do whatever we want with ERC-20 MOVE. For example, we can't mint more of it. 
2. We also can't mint more of the L2 token.